server.js

```diff
@@ -1,16 +1,53 @@
 'use strict';
 
+// ==== packages ====
 const express = require('express');
 
 const cors = require('cors');
 
 require('dotenv').config();
 
-const app  = express();
-app.use(cors());
+// ==== setup the application (server) ====
+const app  = express(); // creates a server from the express library
+app.use(cors()); // app.use loads middleware - we are loading cors so that requests don't get blocked when they are local
 
```

Comment:
More info/way to improve: feel free to use comments more liberally to explain what your code is doing!